apps/frontend/lib/calculate.ts

```
@@ -1,67 +1,91 @@
+        case "/":
+          if (b === 0) {
+            span.end();
+            return NaN;
+          } // Handle division by zero gracefully
+          span.end();
+          return a / b;
+        case "^":
+          span.end();
+          return Math.pow(a, b);
+        // For unsupported operations, return NaN instead of throwing
+        case "**":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)